### DISCUSSION TITLE
Netbox API Tokens don't authenticate API Endpoints

### DISCUSSION THREAD

**Discussion by Sejo9** [mattieserver/netbox-topology-views#421] [Q&A]
Netbox Version 3.6.3
Netbox Topology views 3.8.0

I have tried accessing the api endpoints provided by the plugin from a python script and from postman both including an API token generated in Netbox but on both accounts the token seems to not work and instead I get returned the netbox login html page. Has anyone else experienced this or is there a solution I'm not aware of? Any help will be appreciated.

@mattieserver @dreng

**Comment by mattieserver** [comment]
Does it work from the swagger ui? ( I think it called api docs or something like that in the footer of netbox)

**Reply by Sejo9** [reply]
It works from the swagger ui but only because the request passes a cookie as part of the request for authentication, which will not be the case for a Postman or Python get request. This can be checked by passing the wrong value for the tokenAuth in the Swagger UI, and still the request will go through since it is using the login details provided earlier through the Netbox UI.

**Reply by Sejo9** [reply]
@mattieserver Any updates regarding this issue?
